docs/_docs/04_reference/cli-options.md

```diff
@@ -381,28 +381,44 @@ This should allow you to run `import` on the module and work around those Terraf
 Terragrunt forwards all options to Terraform. The only exceptions are `--version` and arguments that start with the
 prefix `--terragrunt-` (e.g., `--terragrunt-config`). The currently available options are:
 
-- [terragrunt-config](#terragrunt-config)
-- [terragrunt-tfpath](#terragrunt-tfpath)
-- [terragrunt-no-auto-init](#terragrunt-no-auto-init)
-- [terragrunt-no-auto-retry](#terragrunt-no-auto-retry)
-- [terragrunt-non-interactive](#terragrunt-non-interactive)
-- [terragrunt-working-dir](#terragrunt-working-dir)
-- [terragrunt-download-dir](#terragrunt-download-dir)
-- [terragrunt-source](#terragrunt-source)
-- [terragrunt-source-update](#terragrunt-source-update)
-- [terragrunt-ignore-dependency-errors](#terragrunt-ignore-dependency-errors)
-- [terragrunt-iam-role](#terragrunt-iam-role)
-- [terragrunt-exclude-dir](#terragrunt-exclude-dir)
-- [terragrunt-include-dir](#terragrunt-include-dir)
-- [terragrunt-strict-include](#terragrunt-strict-include)
-- [terragrunt-ignore-dependency-order](#terragrunt-ignore-dependency-order)
-- [terragrunt-ignore-external-dependencies](#terragrunt-ignore-external-dependencies)
-- [terragrunt-include-external-dependencies](#terragrunt-include-external-dependencies)
-- [terragrunt-parallelism](#terragrunt-parallelism)
-- [terragrunt-debug](#terragrunt-debug)
-- [terragrunt-check](#terragrunt-check)
-- [terragrunt-hclfmt-file](#terragrunt-hclfmt-file)
-- [terragrunt-override-attr](#terragrunt-override-attr)
+- [CLI commands](#cli-commands)
```

Comment:
This is quite a large diff. Was your branch out of date or something?

Reply:
Looks like my markdown linter went a bit haywire here. My branch wasn't that out of date (just 1 commit behind HEAD)! I'll fix that :)